fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/InsertIntoTableCommand.java

```diff
@@ -123,6 +125,11 @@ public void run(ConnectContext ctx, StmtExecutor executor) throws Exception {
                 ctx.getExecTimeout(),
                 ctx.getSessionVariable().getSendBatchParallelism(), false, false);
 
+        if (((UnboundOlapTableSink) logicalQuery).isPartialUpdate()) {
+            HashSet<String> partialUpdateCols = new HashSet<String>();
+            partialUpdateCols.addAll(((UnboundOlapTableSink) logicalQuery).getColNames());
+            sink.setPartialUpdateInputColumns(true, partialUpdateCols);
+        }
```

Comment:
u should add `isPartialUpdate` to LogicalOlapTableSink and PhysicalOlapTableSink. pass `isPartialUpdate` from unboundSink to LogicalOlapTableSink in `BindInsertTargetTable` pass it from logical to physical in `LogicalOlapTableSinkToPhysicalOlapTableSink` and setPartialUpdateInputColumns to `OlapTableSink` in `org.apache.doris.nereids.glue.translator.PhysicalPlanTranslator#visitPhysicalOlapTableSink`